lib/converters/v2.0.0/converter-v2-to-v1.js

```diff
@@ -405,6 +405,22 @@ _.assign(Builders.prototype, {
         });
     },
 
+    /**
+     * Creates an ordering field for responses of a V2 Item.
+     *
+     * @param {Object} item - The v2 item to extract saved responses from.
+     * @returns {Object[]} - The order of responses within the V2 Item.
+     */
+    responses_order: function (item) {
+        var allResponses = item.responses || item.response;
```

Comment:
Do we have this disparity?

Reply:
I'm not sure, but `responses` are mapped in this manner, and out of an abundance of precaution, I've maintained it